test/asynctest.c

```diff
@@ -123,6 +123,43 @@ static int test_ASYNC_init_thread(void)
     return 1;
 }
 
+static int test_callback(void *arg)
+{
+    printf("callback test pass\n");
+    return 1;
+}
+
+static int test_ASYNC_callback_status(void)
```

Comment:
Aside: Strange - this file does not seem to have been converted to the new test framework. I wonder why it was missed? Anyway this isn't your problem to solve.

Reply:
I can change this code after the new test framework is applied